sdk/iot/provisioning/samples/src/generate_certificate.sh

```diff
@@ -0,0 +1,13 @@
+openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
+openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
```

Comment:
will the `"/CN={}"` change?

Reply:
More work to come in future PRs to have this script print out detailed steps that one needs to take after running it (i.e. link to portal, create DPS registration, use the reg name, etc.)
My plan here: extract into a default variable, override with a parameter.